File: 10- রানরেট ১.c
#include<stdio.h>
int main()
{

    int t;
    double tr,cr,lb;
    scanf("%d",&t);
    for(int i=0;i<t;i++)
    {

        scanf("%lf%lf%lf",&tr,&cr,&lb);

        double co = (300-lb)/6.0;
        double crr = cr/co;

        double rr;

        if(cr>(tr+1))
            rr = 0;
        else
            rr = (tr+1)-cr;

        double lo = lb/6.0;

        double rrr = rr/lo;

        printf("%0.2lf %0.2lf\n",crr,rrr);


    }
    return 0;


}

File: 10- রানরেট ১.py
t = int(input())

for i in range(t):
    a,b,c = [int(x) for x in input().split()]
    balled = 300-c
    if balled == 0:
        crr = 0
    else:
        crr = (b/balled)*6
    rr = (a-b)+1
    if c==0 or b>a:
        rrr = 0
    else:
        rrr = (rr/c)*6

    print("{:.2f}".format(crr),"{:.2f}".format(rrr))
